core/templates/components/forms/custom-forms-directives/edit-thumbnail-modal.template.html

```diff
@@ -5,21 +5,31 @@ <h3>Upload Thumbnail</h3>
 <div class="modal-body" style="min-height: 300px;">
   <div class="oppia-thumbnail-uploader">
     <div ng-hide="uploadedImage">
+      <strong>Please upload an SVG image.</strong>
+      <div class="alert alert-warning">
+        <span>Here are some guidelines to follow:</span>
+        <ul>
+          <li>Uploaded image should have an aspect ratio of 4:3 or 16:9.</li>
```

Comment:
Shouldn't you be showing a specific aspect ratio depending on the context? I'm pretty sure that, per the thumbnails guidelines, different contexts need a different aspect ratio.